webpack.config.js

```diff
@@ -1,4 +1,4 @@
-const taskNumber = '';
+const taskNumber = 05;
```

Comment:
Gdybyś wstawiła jako string tj `'01'` to byś nie musiała zmieniać nazw folderów ;)